test/PerformanceTests/SerializationComparisonsTests/JsonWriterBenchmarks/Benchmarks.cs

```diff
@@ -65,29 +71,30 @@ public async Task WriteToMemoryAsync()
         public async Task WriteToFileAsync()
         {
             // multiple writes to increase benchmark duration
-            await WritePayloadAsync();
-            await WritePayloadAsync();
-            await WritePayloadAsync();
-            await WritePayloadAsync();
-            await WritePayloadAsync();
+            await WritePayloadAsync(data);
+            await WritePayloadAsync(data);
         }
 
         [Benchmark]
         [BenchmarkCategory("RawValues")]
         public async Task WriteWithRawValues()
         {
             // multiple writes to increase benchmark duration
-            await WritePayloadAsync(includeRawValues: true);
-            await WritePayloadAsync(includeRawValues: true);
-            await WritePayloadAsync(includeRawValues: true);
-            await WritePayloadAsync(includeRawValues: true);
-            await WritePayloadAsync(includeRawValues: true);
+            await WritePayloadAsync(data, includeRawValues: true);
+            await WritePayloadAsync(data, includeRawValues: true);
         }
 
-        private async Task WritePayloadAsync(bool includeRawValues = false)
+        [Benchmark]
+        [BenchmarkCategory("ToFile")]
+        public async Task WriteToFileWithLargeValuesAsync()
+        {
+            await WritePayloadAsync(dataWithLargeValues);
+        }
+
+        private async Task WritePayloadAsync(IEnumerable<Customer> payload, bool includeRawValues = false)
         {
-            using var outputStream = new FileStream(filePath, FileMode.Create, FileAccess.Write, share: FileShare.ReadWrite);
-            await writer.WritePayloadAsync(data, outputStream, includeRawValues);
+            //using var outputStream = new FileStream(filePath, FileMode.Create, FileAccess.Write, share: FileShare.ReadWrite);

```

Comment:
Why did you need to comment this statement instead of deleting it?

Reply:
Oversight. Thanks.